src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/DotNetCoreProjectCompatibilityDetector.cs

```diff
@@ -98,7 +98,8 @@ public async Task InitializeAsync()
             // Check to see if a solution is already open. If so we set _solutionOpened to true so that subsequent projects added to 
             // this solution are processed.
             if (ErrorHandler.Succeeded(solution.GetProperty((int)__VSPROPID4.VSPROPID_IsSolutionFullyLoaded, out object isFullyLoaded)) &&
-                isFullyLoaded is true)
```

Comment:
I didn't know that boxed.  That's too bad.

Reply:
Yeah it could possibly generate better code, but it's not a biggie. Might be nice to have a `System.Boolean.TrueObject`/`FalseObject` for these kinds of comparisons.